01-html/01-koajs.html

```diff
@@ -1,4 +1,671 @@
 <!-- http://koajs.com/
 Don't worry about all the content - it's an awfully long page.
 I picked this because it's a pretty simple layout.
- -->
\ No newline at end of file
+ -->
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Koa - next generation web framework for node.js</title>
+</head>
+<body>
+  <header>
+    <h1>koa</h1>
+    <h5>next generation web framework for node.js</h5>
+    <nav>
+      <ul>
+        <button type="button">contents</button>
+        <li><a href="introduction">introduction</a></li>
+        <li><a href="application">application</a></li>
+        <li><a href="context">context</a></li>
+        <li><a href="request">request</a></li>
+        <li><a href="response">response</a></li>
+        <li><a href="links">links</a></li>
+      </ul>
+    </nav>
+  </header>
+  <hgroup>
```

Comment:
`<hgroup>` isn't used terribly often, but when it is, it is generally for grouping together a section of headers and subheaders. For example, it would be appropriate to wrap lines 13 & 14 in `<hgroup>`.